### DISCUSSION TITLE
Welcome Contributors!

### DISCUSSION THREAD

**Discussion by MatthewJeffson** [Seeed-Studio/wiki-documents#339] [Announcements]
## The progress of Seeed Studio wiki platform update is publicly available
- We are dedicated to enhancing the user experience and providing better support through the development of our wiki platform. 
- Our focus extends beyond product usage and also covers IoT, AI, TinyML, industrial computing, and other related fields. 
- This platform will keep you informed about all updates to the wiki platform, including new content related to product introductions and usage, as well as any new website functions that are added.
- Your contribution is essential to us! We really value your input and would greatly appreciate your assistance in generating ideas. 
## Join forces with us!
- We really appreciate if you can [contribute in any way](https://github.com/orgs/Seeed-Studio/projects/6/views/1), suggesting updates, fixing typos, accepting the assignments in demand, etc
- Submit a "Pull Request" and accept our GitHub invitation, to become one of our contributor.
- You can also come and [Share Your Own Ideas](https://github.com/Seeed-Studio/wiki-documents/issues/322) you may have with us.
## How do we work with contributors?
We welcome contributions from all individuals who wish to help us improve our wiki platform. 
1. We have a range of assignments available to enhance the wiki contents for contributors, each with detailed description.
2. The contributors can submit their content via "Pull Request" when the assignments have been finished.
3. The maintainers will merge the submission and record the contributions.

We believe that collaboration is key to success and we look forward to working with our contributors to make our platform the best it can be.

## What do we offer?
We want to express our gratitude to our contributors by offering a range of rewards.  Depending on the difficulties(Tier 0/1/2/3), the time we expect(Days 1/3/7/15),  the actual  submission time and the actual submission contents, we will offer different levels of rewards.
1. We will provide our products or cash bonus to our contributors as a token of appreciation.
2. For those contributors who join our "Ranger Program"(announced soon), we offer even more exciting rewards, including access to our prototype products.
3. In addition, all contributors and their contributions will be acknowledged and showcased on our wiki platform.

We believe that our contributors are an integral part of our success, and we want to ensure that they are recognized and rewarded for their contributions.

**Comment by vongomben** [comment]
Hi! 
As a Ranger or applying wanna-be one, I miss the products!
How do I apply for: 
* XIAO ESP32C3
* reTerminal DM (Ihave one reTerminal though)
* SenseCAP Indicator

I don't need all of them, even one will do. If I hvae to pick one maybe the Node-red things around reTerminal could be a starting point for issues. Also  testing the svg nodes for reating custom interfaces on the reTerminal would an interesting test / documentation / testing case.

**Reply by 33Alison** [reply]
Hey, it's great to hear that you're excited about the Ranger Program! We're still in the process of finalizing all the program details, but don't worry, we'll be sharing more information with you next week after our three-day Dragon Boat Festival holiday. And if you have any suggestions or advice for the Ranger Program, we're open to fantastic ideas!

**Reply by MatthewJeffson** [reply]
Hi! Welcome Ben!
We currently are at the test stage and the main progress should be like: 
1. The ranger accepts the assignment
2. We send the sample
3. follow up discussion...
Would you mind share what assignments you want to accept and then we can discuss how to send you the products.
Regards!

**Reply by 256ericpan** [reply]
@MatthewJeffson please send out a Xiao C3 and S3, indicators for Davide to start. It's already taking too long.

**Reply by MatthewJeffson** [reply]
@vongomben Hi! May I say that getting our products first and then considering which task to accept is a satisfactory way of communication for you? Because currently I haven't seen some comments with names under the assignments like [here](https://github.com/Seeed-Studio/wiki-documents/issues/334).
I have already sent an email to your Google mailbox, regarding the address where we can send the products to you and other issues we can go through.:D
I am terribly sorry for long time delay. Hope we can have a pleasant cooperation later!
Thank you!

**Reply by vongomben** [reply]
Hi @MatthewJeffson
Thanks for your invitation and reply. 
I'll try my best to search and apply for issues. 
Answered you for the address privately. 

Thanks!

**Reply by MatthewJeffson** [reply]
Thank you! I have received the email and will arrange for products to be delivered as soon as possible!
Have a nice day!
